phantazm-api/src/main/java/com/github/phantazmnetwork/api/inventory/BasicInventoryObject.java

```diff
@@ -8,7 +8,6 @@
 /**
  * Basic implementation of an {@link InventoryObject}.
  */
-@SuppressWarnings("ClassCanBeRecord")
```

Comment:
unrelated to the PR but thank Notch for whatever intelliJ update caused these warnings to not appear on every hecking class without non-final fields